lib/src/backend.rs

```diff
@@ -217,6 +217,11 @@ impl CopyRecords {
     pub fn for_target(&self, target: &RepoPath) -> Option<&CopyRecord> {
         self.targets.get(target).and_then(|&i| self.records.get(i))
     }
+
+    /// Gets all copy records.
+    pub fn records(&self) -> impl Iterator<Item = &CopyRecord> + '_ {
+        self.records.iter()
+    }
```

Comment:
Nit: This should just be `records_iter` as reading `copy_records.records()` isn't really pleasant.

Reply:
went with just `iter()`